ttnn/cpp/ttnn/operations/core/to_layout/to_layout_op.cpp

```diff
@@ -23,6 +23,18 @@ namespace core {
 
 namespace detail {
 
+inline bool nd_support(const ttnn::Tensor& tensor_arg, const ttnn::Layout layout) {
```

Comment:
Can you please give it a more clear name?
This function is something like validate_nd_support right?